src/test/java/org/testng/internal/DynamicGraphTest.java

```diff
@@ -240,6 +242,22 @@ public void testOrderingOfEdgesWithSameWeight() {
     expected = Arrays.asList("TestNG1.test3TestNG1", "TestNG2.test3TestNG2", "TestNG3.test3TestNG3");
     runAssertion(graph, expected);
   }
+  
+  @Test
+  public void edgeWeightTest1() {
```

Comment:
Can I ask to follow the new test convention (not documented and not the one of the file):
 Instead of static vars, use `InvokedMethodNameListener`.